### PR TITLE
FEI-4271: Fix __vite__injectQuery() issue for real

### PR DESCRIPTION
**Pull request by k4b7**
Developers have been experiencing the occasional error where `__vite__injectQuery()` inside an import path string.  This only happens about once out of every 5-10 restarts of vite.  I narrowed the issue down to `es-module-lexer` sometimes providing wonky data for one of the imports in its parse results.  This PR detects when this occurs and works around the issue by reparsing the import statement using a regex.  This only seems to happen for static imports so that's all the fix handles.

## Test Plan:
- pnpm install
- pnpm run ci-build-vite
- pnpm run test-serve -- packages/vite

TODO:
- pnpm package
- copy/paste the package into webapp in /services/static/third_party
- update services/static/package.json to point the "vite" dependency to this new file

### Notes:
- When I ran the full test suite locally I saw a bunch of errors in other packages.  I'll check out the parent branch and try running the full test suite there as well to see if there are any differences
- Running the full test suite result in node running out of memory
- When trying to make a commit, I ran into the following error `Error: Cannot find module './verifyCommit.ts'` as part of some pre-commit check.  I used `-n` with `git` to disable the check.

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -83,7 +83,7 @@ importers:
       create-vite: latest
       kolorist: ^1.5.0
     dependencies:
-      create-vite: link:../create-vite
```
Comment (jeresig) [inline pnpm-lock.yaml]: Should this file have changed?

**Review by jeresig — APPROVED** [review]
Huh... I wish we knew why it was failing 1 glad it seems to be fixing the issue, at least!

File: pnpm-lock.yaml (hunk 1 above)
Comment (k4b7) [reply pnpm-lock.yaml]: These lines were changed automatically as a result of running `pnpm install`.

Comment (jeresig) [reply pnpm-lock.yaml]: I wonder if that's intended though? I wonder if there's a frozen lock file equivalent to what we use for yarn. Probably worth reverting the change!

**Pull request merged**